store/index.js

```diff
@@ -240,6 +241,44 @@ export const getters = {
         return p_state.transformationHeuristics[columnDataType];
     },
 
+    getUniqueValues: (p_state) => (p_category, p_maxValues="None") => {
+
+        // 1. Construct a list of unique values for each column
```

Comment:
I think this should be `Construct an object containing...` since `uniqueValues` is an object

Reply:
Good point. Comment needed to be updated with implementation change.